app/controllers/application_controller.rb

```diff
@@ -17,4 +17,27 @@ def set_locale
   def default_url_options
     {locale: I18n.locale}
   end
+
+  def check_amin
+    redirect_to root_path unless current_user.role?
```

Comment:
thêm message vào để thân theienj ng dùng cu, tự nhiên đuổi họ sang root mà ko 1 lý do, chia tay cũng có lý do mà :v 

ngoài ra, `current_user.role?` thành `current_user.admin?` dc ko em
nếu field role em đang dùng enum in rails